scripts/v2xhub_log_analysis_erv.py

```diff
@@ -25,6 +25,52 @@ def get_filenames():
 
 
 def process_input_log_file(inputfile, search_keyword):
+<<<<<<< HEAD
+    fields_dict = {}
+    fields_dict["Time (UTC)"] = []
+    try:
+        file_stream = open(inputfile, 'r')
+        while True:
+            line = file_stream.readline()
+            # if line is empty, end of file is reached
+            if not line:
+                break
+            if len(line.strip()) == 0:
+                continue
+            if len(line.strip().split(' - ')) < 2:
+                continue
+
+            txt = line.strip().split(' - ')[1]
+            txt_list = [x.strip() for x in  txt.strip().split(":") if x.strip() != 'INFO' if x.strip() != 'DEBUG' if x.strip() != 'ERROR' ]
+            # Look for the specific metric by keyword
+            if len(txt_list) > 1 and search_keyword.lower().strip() == txt_list[0].strip().lower():
+                metadata_list = [x for x in line.strip().split(
+                    ' - ')[0].split("[") if x != '']
+                time = ''.join(metadata_list).split(']')[0].replace('"','').replace('log','').replace('{','').replace('{}','').replace('}','')
+                metric_field_value = ''
+                metric_field_title = ''            
+                metric_field_title = txt_list[0].strip().replace("stream","").replace('\\n','').replace('"','').replace(',','')
+                metric_field_value = txt_list[1].strip().replace("stream","").replace('\\n','').replace('"','').replace(',','').replace('\\u003e','>').replace('\\u003c','<').replace('\\','"')
+                # The actual BSM XML content from the log is on the next line
+                if metric_field_title.strip() in ['Incoming BSM is not from Emergency Response Vehicle (ERV)', 'Receive BSM'] :
+                    metric_field_value = file_stream.__next__()
+                    metric_field_value = metric_field_value.strip().replace("stream","").replace('\\n','').replace('"','').replace(',','').replace('\\u003e','>').replace('\\u003c','<').replace('\\','"')
+                    metric_field_value_list = metric_field_value.split(':')
+                    for value in metric_field_value_list:
+                        if '<BasicSafetyMessage>' in value:
+                            metric_field_value = value
+                elif metric_field_title.strip() in ['Successfully forwarded message to cloud via curl', 'Forwarding message to cloud via curl']:
+                    if '<BSMRequest>' not in metric_field_value:                   
+                        continue
+                if metric_field_title not in fields_dict.keys():
+                    fields_dict[metric_field_title] = []                
+                fields_dict[metric_field_title].append(metric_field_value)
+                fields_dict["Time (UTC)"].append(time.replace('status:200',''))            
+        file_stream.close()
+    except FileNotFoundError:
+        print(f'Log file not found {inputfile}')
+        exit()
+=======
```

Comment:
It looks like in several places in this PR you still have the merge conflict delimiters.

Reply:
https://github.com/usdot-fhwa-OPS/V2X-Hub/commit/354ef421229caf43b9f18dec541cc4e3ded3384d